JetAnalyzers/src/JetResponseAnalyzer.cc

```diff
@@ -115,6 +115,50 @@ void JetResponseAnalyzer::beginJob()
 }
 
 
+
+
+
+
+void getMult( vector<reco::CandidatePtr> const & particles, int* nMult, int* chMult ) {
+
+  vector<reco::CandidatePtr>::const_iterator itParticle;
+  for (itParticle=particles.begin();itParticle!=particles.end();++itParticle){
+    const reco::Candidate* pfCand = itParticle->get();
+
+    switch (std::abs(pfCand->pdgId())) {
+
+      case 211: //PFCandidate::h:       // charged hadron
```

Comment:
Don't use unnamed constants. Instead, you should define enums as in https://github.com/cms-jet/JetMETAnalysis/blob/9ea9753d8a475210644c6efaa224a5c0b8dd331f/JetUtilities/interface/JRAEvent.h#L37 and use those as your cases.

You also only really have two cases:
1. You add one to the `nMult` variable
2. You add one to the `chMult` variable
You should combine these cases using and OR of the conditions inside if, else if, else statements. This will reduce the number of comparisons. You can use the `std::any_of` function to reduce the verbosity of the conditional (https://www.cplusplus.com/reference/algorithm/any_of/).